R/layer.R

```diff
@@ -463,8 +463,6 @@ Layer <- ggproto("Layer", NULL,
   }
 )
 
-is.layer <- function(x) inherits(x, "Layer")
```

Comment:
I'd actually say we should export this rather than remove it, since we do so for all the other main classes